components/dashboard/src/onboarding/StepOrgInfo.tsx

```diff
@@ -0,0 +1,180 @@
+    const isValid = jobRoleError.isValid && signupGoals.length > 0;
+
+    return (
+        <OnboardingStep
+            title="Tell us more about you"
+            subtitle="Let us know what brought you here."
+            error={updateUser.isError ? "There was a problem saving your answers" : ""}
+            isValid={isValid}
+            isLoading={updateUser.isLoading}
+            onSubmit={handleSubmit}
+        >
+            <SelectInputField
+                value={jobRole}
+                label="I work in..."
+                onChange={(val) => {
+                    if (val !== "other") {
+                        setJobRoleOther("");
+                    }
+                    setJobRole(val);
+                }}
+                hint={
+                    jobRole !== JOB_ROLE_OTHER
+                        ? "Please select the role that best describes the type of work you'll use Gitpod for"
+                        : ""
+                }
+                error={jobRoleError.message}
+                onBlur={jobRoleError.onBlur}
+            >
+                {jobRoleOptions.map((o) => (
+                    <option key={o.value} value={o.value}>
+                        {o.label}
+                    </option>
+                ))}
+            </SelectInputField>
+
+            {jobRole === JOB_ROLE_OTHER && (
+                <TextInputField
+                    value={jobRoleOther}
+                    onChange={setJobRoleOther}
+                    placeholder="Please specify"
+                    hint={
+                        jobRole === JOB_ROLE_OTHER
+                            ? "Please select the role that best describes the type of work you'll use Gitpod for"
+                            : ""
+                    }
+                />
+            )}
+
+            <TextInputField
+                value={companyWebsite}
+                label="Company Website (optional)"
+                type="url"
+                placeholder="https://"
+                onChange={setCompanyWebsite}
+            />
+
+            <InputField label="I'm signing up for Gitpod to..." />
+            <div className="mt-4 ml-2 space-y-2">
+                {signupGoalsOptions.map((o) => (
+                    <div key={o.value} className="flex space-x-2 justify-start items-center">
+                        <input
+                            type="checkbox"
+                            className="rounded"
+                            value={o.value}
+                            id={`goals_${o.value}`}
+                            checked={signupGoals.includes(o.value)}
+                            onChange={(e) => {
+                                if (e.target.checked) {
+                                    addSignupGoal(o.value);
+                                } else {
+                                    removeSignupGoal(o.value);
+                                }
+                            }}
```

Comment:
Some Code cutting Suggestion 😉 
```suggestion
                            onChange={(e) => {
                                e.target.checked ? addSignupGoal(o.value) : removeSignupGoal(o.value);
                            }}
```